.github/scripts/test-info.mjs

```diff
@@ -82,6 +82,20 @@ const components = {
             'AzureBlobStorageQueue',
         ],
     },
+    'bindings.aws.s3': {
```

Comment:
Something seems off here.

For the Docker (localstack) case it should say `certification: false` and also `requireAWSCredentials: false` and `requireTerraform: false`

You you should only run local stack for conformance test.

Of course you will also run the terraform conformance test with regular AWS. Certification is AWS + Terraform only.

In general I highly recommend you refactor `bindings.aws.s3` to `bindings.aws.s3.localstack` to make it clear what is what. I have seen strange issues with nested tests executing all the tests. It is best if these are in sibling folders.